docs/data/destinations/meta-pixel.md

```diff
@@ -31,10 +33,69 @@ To configure an Event Streaming integration from Amplitude to Meta Pixel, you ne
 2. Scroll down to Conversions API and click Generate access token in Set up manually.
 3. Copy the Pixel ID at the top of the page and the newly generated access token.
 
-### Amplitude setup 
+### Amplitude setup
+
+1. In Amplitude, navigate to **Data**.
+2. On the sidebar, under **Connections**, select **Catalog**.
+3. Select the **Destinations** tab.
+4. Select **Event Streaming**.
+5. Click **Meta Pixel - Event Streaming**.
+6. Enter a sync name, then click **Create Sync**.
+
+After you create the destination, you must configure the settings.
+
+#### Configure settings
+
+1. On the **Settings** tab, click **Edit**.
+2. Enter your **Pixel ID**.
+3. Enter your **Conversions API Access Token**.
+4. **Send Events** sends events ingested by Amplitude to Meta.
+      1. To send an event, toggle **Send Events** to **Enabled**.
+      2. Expand the **Select and filter events** panel, and select which events to send.
+      3. To select event properties to send, expand the **Specify event properties to send** panel, and select properties you want to include. If you don't select any properties here, Amplitude doesn't send any.
+5. Save when finished.
+
+After you configure your settings, configure your mappings.
+
+#### Configure mappings (recommended)
+
+For newer versions of the Meta Pixel destination, you can map Amplitude properties to fields in Meta Pixel that will be included in the `user_data` array and used for conversions. If you don't configure mappings, or are using an older version that doesn't support mapping, the default values described below are used instead.
+
+1. Click the **Mappings** tab, then click **Edit**.
+2. Select a **External ID**. This is a unique identifier for users in Meta Pixel. It defaults to Amplitude **User ID** (`user_id`).
+3. Select as many other mappings as possible for best possible conversion results.
```

Comment:
If this is the case, should these mappings be recommended rather than optional?

Reply:
I think they're not quite THAT important lol. The destination has been working fine without them, but including more will be better